Fix base64 function prototypes.  These are used when not provided by the libc.

diff --git a/fdm.h b/fdm.h
index 481b7930e04f..dafc7818a711 100644
--- a/fdm.h
+++ b/fdm.h
@@ -724,8 +724,8 @@ size_t		 strlcat(char *, const char *, size_t);
 
 #ifndef HAVE_B64_NTOP
 /* base64.c */
-int b64_ntop(src, srclength, target, targsize);
-int b64_pton(src, target, targsize);
+int b64_ntop(const u_char *, size_t, char *, size_t);
+int b64_pton(const char *, u_char *, size_t);
 #endif
 
 /* shm.c */
